src/app/modules/shared_models/member_model.nim

```diff
@@ -168,6 +178,18 @@ QtObject:
     self.endInsertRows()
     self.countChanged()
 
+  proc addItems*(self: Model, items: seq[MemberItem]) =
+    let modelIndex = newQModelIndex()
+    defer: modelIndex.delete
+
+    let first = self.items.len
+    let last = first + items.len - 1
+
+    self.beginInsertRows(modelIndex, first, last)
+    self.items.add(items)
+    self.endInsertRows()
+    self.countChanged()
```

Comment:
As we have `ModelCount` attached property, there is no need to expose `count` from nim. Ideally we should just remove it from the model. Not necessary in this PR, just pointing for the future.